orbit/cmd/orbit/orbit.go

```diff
@@ -515,6 +510,90 @@ func main() {
 		}
 		g.Add(flagRunner.Execute, flagRunner.Interrupt)
 
+		trw := token.NewReadWriter(filepath.Join(c.String("root-dir"), "identifier"))
+
+		if err := trw.LoadOrGenerate(); err != nil {
+			return fmt.Errorf("initializing token read writer: %w", err)
+		}
+
+		if orbitClient.GetServerCapabilities().Has(fleet.CapabilityOrbitEndpoints) &&
+			orbitClient.GetServerCapabilities().Has(fleet.CapabilityTokenRotation) {
+			log.Info().Msg("token rotation is enabled")
+
+			// we enable remote updates only if the server supports them by setting
+			// this function.
+			trw.SetRemoteUpdateFunc(func(token string) error {
+				return orbitClient.SetOrUpdateDeviceToken(token)
+			})
+
+			// ensure the token value is written to the remote server, we might have
+			// a token on disk that wasn't written to the server yet
+			if err := trw.Write(trw.GetCached()); err != nil {
```

Comment:
Is this necessary if we have:
```go
// perform an initial check to see if the token
// has not been revoked by the server
if err := deviceClient.CheckToken(trw.GetCached()); err != nil {
  if err := trw.Rotate(); err != nil {
    return fmt.Errorf("rotating token: %w", err)
  }
}
```
below?

Reply:
I think only if we want to reuse the token, so, you're right not necessary. I will remove